websites/Z/ZeroTwo/presence.ts

```diff
@@ -0,0 +1,27 @@
+const presence = new Presence({
+    clientId: "752067809214857298"
+  }),
+  browsingTimestamp = Math.floor(Date.now() / 1000);
+  
+  presence.on("UpdateData", async () => {
+    const presenceData: PresenceData = {
+      largeImageKey: "logo"
+    };
+  
+    if ((await presence.getSetting("incognito")) === false) {
+      presenceData.details = document.getElementById("premid").innerText;
+      if ((await presence.getSetting("showTimestamp")) === true)
+        presenceData.startTimestamp = browsingTimestamp;
+    }
+    if ((await presence.getSetting("buttons")) === true) {
+      presenceData.buttons = [
+        {
+          label: "Invite ZeroTwo",
+          url: "https://zerotwo.wtf/invite"
+        }
+      ];
```

Comment:
Presences that use buttons should follow extra requirements:
Redirects to main page are prohibited.
**Promoting websites by them is prohibited.**
They can't display information you couldn't fit in other fields.
Redirecting directly to audio/video stream is prohibited.